src/context/auth.tsx

```diff
@@ -27,6 +28,17 @@ interface ProviderProps {
 export const AuthProvider: FC<ProviderProps> = ({ children }) => {
   const { data: settings, isLoading } = useSettings();
 
+  const { data: projects = [] } = useQuery({
+    queryKey: [queryKeys.projects, 1],
```

Comment:
nit: maybe assign 1 to a variable (e.g. recursion) to make it clear what the number means?

Reply:
We always fetch with recursion set to 1 to hit the same cached result. So we can as well remove the parameter from the function. I did that together with a little refactoring of the other usages of the function to simplify.

Reply:
perfect, that makes sense :+1: